src/corona_hakab_model/generation/circles_consts.py

```diff
@@ -19,7 +18,7 @@ class CirclesConsts(NamedTuple):
     population_size: int = 20_000
     ages: List[int] = [10, 40, 70]
     age_prob: List[int] = [0.30, 0.45, 0.25]
-    teachers_ratio = 0.015 # ratio of teachers out of workforce
+    teachers_ratio = 0.04 # ratio of teachers out of workforce
```

Comment:
IS this number of teacher that DO go to the work or not? The name is not so indicative

Reply:
This is the number of teachers out of all working adults. They go to school and not to work (connect with teachers and kids and not just other adults)

Reply:
OK, sothis is the percentage of teachers from the working population? It can be easier understood if you refactor the name of the variable (if you cant, then give more meaningfull name to it)